tests/environment-delete.bats

```diff
@@ -0,0 +1,71 @@
+#! /usr/bin/env bats
+# -*- mode: bats; -*-
+# ============================================================================ #
+#
+# Test 'flox delete'
+#
+# ---------------------------------------------------------------------------- #
+
+load test_support.bash
+# bats file_tags=delete
+
+# ---------------------------------------------------------------------------- #
+
+# Helpers for project based tests.
+
+project_setup() {
+  export PROJECT_DIR="${BATS_TEST_TMPDIR?}/test"
+  rm -rf "$PROJECT_DIR"
+  mkdir -p "$PROJECT_DIR"
+  pushd "$PROJECT_DIR" >/dev/null || return
+}
+
+project_teardown() {
+  popd >/dev/null || return
+  rm -rf "${PROJECT_DIR?}"
+  unset PROJECT_DIR
+}
+
+# ---------------------------------------------------------------------------- #
+
+setup() {
+  common_test_setup
+  project_setup
+}
+teardown() {
+  project_teardown
+  common_test_teardown
+}
+
+setup_file() {
+  export FLOX_FEATURES_ENV=rust
+}
+
+dot_flox_exists() {
+  # Since the return value is based on the exit code of `test`:
+  # 0 = true
+  # 1 = false
+  [[ -d .flox ]];
+  echo "$?"
+}
```

Comment:
This is not intuitive at all

Either rename the function to `dot_flox_absent` or run 

```
dot_flox_exists() {
  # Since the return value is based on the exit code of `test`:
  # 0 = true
  # 1 = false
  [[ -d .flox ]];
}

@test "..." {
 run dot_flox_exists;
 assert_success;
}
```

or at that point even 

```
run "$FLOX_CLI" envs;
assert_success; 
```